windows/src/engine/keyman32/appint/appint.cpp

```diff
@@ -203,3 +203,37 @@ AppIntegration::AppIntegration()
 	hwnd = NULL;
   FShiftFlags = 0;
 }
+
+BOOL ContextItemsFromAppContext(WCHAR const* buf, km_kbp_context_item** outPtr)
+{
+
+  km_kbp_context_item* context_items  = new km_kbp_context_item[wcslen(buf) + 1];
+  // would like to check bad allocation but nothrow not known to this compiler
```

Comment:
As far as I know, we have never received an out-of-memory error in the 20+ years of running Keyman :grin:  So may be something we don't really need to worry about too much!

Reply:
removed comment